FLEDGE_browser_bidding_and_auction_API.md

```diff
@@ -130,6 +149,47 @@ const myAuctionConfig = {
 Note that since the `serverResponse` field in the config is a promise it is
 possible to run the on-device auction in parallel with the B&A auction.
 
+## Request Size and Configuration
+
+### Request Size Controls
+The `requestSize` field provided to `navigator.getInterestGroupAdAuctionData()`
+can be used to specify the maximum size of the returned request. If the
+`perBuyerConfig` field is not present (or is empty) then the `requestSize` field
+sets the maximum size for the returned encrypted blob. If the blob fits into a
+size bucket smaller than `requestSize` then that size will be used instead.
+
+If the `perBuyerConfig` field is specified and non-empty, the returned encrypted
+blob will be exactly `requestSize` bytes long unless there was an error. If an error
```

Comment:
As a seller concerned about request size, `perBuyerConfig` returning the exact `requestSize` is quite suboptimal. While it's highly desirable to fairly fit multiple buyers on the request, we would prefer to take the min(actual_size, request_size) instead.

Could we satisfy the privacy constrain in a way that better optimizes for streamlined request payload size?

Reply:
I agree that returning the exact request size is unfortunate, but returning even a bucketed size leaks significantly more data than we are comfortable with.

One alternative that we considered was allowing the config to be specified out of band  per seller, and only allowed to change relatively slowly (preventing the leak from calling the API multiple times with the slightly different parameters). Unfortunately this can be worked around using multiple sellers that have configs that are slightly different. The sellers don't even have to be cooperating because the attack can be completely client-side.